requirements.txt

```diff
@@ -0,0 +1,2 @@
+# Make sure clams-python version is explicitly specified, at least the lower bound
+clams-python==1.2.2
```

Comment:
1. Please consider update to clams-python 1.2.4, That will come with `cli.py` by default when you do `clams develop` command 
2. Please provide the list of dependencies. Our convention: strict pin the `clams-python` and pin lower bound for other libraries